integration-cli/docker_cli_links_test.go

```diff
@@ -25,13 +23,8 @@ func (s *DockerSuite) TestLinksInvalidContainerTarget(c *check.C) {
 	testRequires(c, DaemonIsLinux)
 	out, _, err := dockerCmdWithError("run", "--link", "bogus:alias", "busybox", "true")
 
-	if err == nil {
-		c.Fatal("an invalid container target should produce an error")
-	}
-	if !strings.Contains(out, "Could not get container") {
-		c.Fatalf("error output expected 'Could not get container', but got %q instead; err: %v", out, err)
-	}
-
+	c.Assert(err, check.NotNil, check.Commentf("an invalid container target should produce an error"))
```

Comment:
this can be just a regular comment 
`// an invalid container target should produce an error`
on the same line or the previous line, and it will be printed out.